Irina/src/main/java/oop/Address.java

```diff
@@ -0,0 +1,42 @@
+package oop;
+
+public class Address {
+    private String country;
+    private String city;
+    private String street;
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public void setStreet(String street) {
+        this.street = street;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public String getStreet() {
+        return street;
+    }
+
+    public Address(String country, String city, String street) {
+        this.country = country;
+        this.city = city;
+        this.street = street;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public String getFullAddress(String address) {
+        address = country + ", " + city + ", " + street;
+        return address;
```

Comment:
You can simplify this by writing directly **return address + ...**

Reply:
Done